staging/com.smartnews/dataset.tt

```diff
@@ -0,0 +1,90 @@
+dataset @com.smartnews language "en" {
+    query := @com.smartnews.top_articles()
+    #_[utterances=["smartnews",
+                   "smart news",
+                   "smartnews top",
+                   "smart news top",
+                   "news",
+                   "top news",
+                   "top smartnews",
+                   "top smart news",
+                   "news from smartnews",
+                   "news on smartnews",
+                   "breaking news",
+                   "smart news",
+                   "today news",
+                   "popular news",
+                   "important news",
+                   "today's news",
+                   "global news",
+                   "hot topic",
+                   "hot news",
+                   "high rank news"]]
```

Comment:
You should include phrases with the words "article", "story", "news story", "news report", and similar.